frontend/package-lock.json

```diff
@@ -2530,13 +2624,13 @@
         "color-name": {
           "version": "1.1.3",
           "resolved": "https://registry.npmjs.org/color-name/-/color-name-1.1.3.tgz",
-          "integrity": "sha512-72fSenhMw2HZMTVHeCA9KCmpEIbzWiQsjN+BHcBbS9vr1mtt+vJjPdksIBNUmKAW8TFUDPJK5SUU3QhE9NEXDw==",
+          "integrity": "sha1-p9BVi9icQveV3UIyj3QIMcpTvCU=",
```

Comment:
Please delete your NPM dependencies (`rm -rf node_modules`), reset your lock file to the branch and re-install.

Reply:
@andrewballantyne I tried, I even clear the npm cache but it doesn't work...

Reply:
This is a problem. Leave it for now -- we can clean this up later 😞

Reply:
Yeah I guess it's probably also related to my npm version.